spec/hash-map.dd

```diff
@@ -29,6 +31,17 @@ $(HEADERNAV_TOC)
         of the keys inserted into the array. In particular, in a $(D foreach) loop the
         order in which the elements are iterated is typically unspecified.)
 
+$(H2 $(LNAME2 literals, Literals))
+
+$(SPEC_RUNNABLE_EXAMPLE_RUN
+---
+auto aa = [21u:"he", 38:"ho", 2:"hi"];
```

Comment:
Style guide doesn't specify this, but I think there should be a space after the `:`